src/main/java/com/rybka/view/ExchangeView.java

```diff
@@ -28,9 +32,14 @@ public void showDialog() {
             System.out.print("Enter value: ");
             var userValue = scanner.nextDouble();
             System.out.print("Insert target currency: ");
-
             var userTargetCurrency = scanner.next().toUpperCase();
 
+            int userExportChoice;
+            do {
+                System.out.print("Choose way to export data (1-Console, 2-CSV file, 3-Json file): ");
```

Comment:
Why did you add this code? Take a look at task description.

Reply:
fixed